app/views/admin/claims/search.html.erb

```diff
@@ -9,14 +9,42 @@
 
     <%= form_with url: search_admin_claims_path, method: :get do |form| %>
       <div class="govuk-form-group">
-        <%= form.label :reference, "Enter a reference", class: "govuk-label" %>
-        <%= form.text_field :reference, class: "govuk-input" %>
+        <%= form.label :query, "Enter a reference, email address, Teacher Reference Number, or surname", class: "govuk-label" %>
+        <%= form.text_field :query, class: "govuk-input" %>
       </div>
 
       <div class="govuk-form-group">
         <%= form.submit "Search", class: "govuk-button" %>
       </div>
     <% end %>
 
+    <% if @claims && @claims.any? %>
+
+      <table class="govuk-table">
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">Reference</th>
+            <th scope="col" class="govuk-table__header">Applicant Name</th>
+            <th scope="col" class="govuk-table__header">Service</th>
+            <th scope="col" class="govuk-table__header">Academic Year</th>
+            <th scope="col" class="govuk-table__header"><span class="govuk-visually-hidden">Actions</span></th>
+          </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+          <% @claims.each do |claim| %>
+            <tr class="govuk-table__row">
+              <th scope="row" class="govuk-table__header"><%= claim.reference %></th>
+              <td class="govuk-table__cell"><%= claim.full_name %></td>
+              <td class="govuk-table__cell"><%= claim.policy.short_name %></td>
+              <td class="govuk-table__cell"><%= claim.academic_year %></td>
+              <td class="govuk-table__cell">
+                <%= link_to 'View tasks', admin_claim_tasks_path(claim), class: "govuk-link" %>
```

Comment:
I wonder if it makes sense to use different words here because we're no longer listing just claims that are awaiting checking:

```suggestion
                <%= link_to "View details"', admin_claim_tasks_path(claim), class: "govuk-link" %>
```